hack/generator/pkg/astmodel/package_reference.go

```diff
@@ -36,3 +44,219 @@ func PackageAsLocalPackage(pkg PackageReference) (LocalPackageReference, error)
 	}
 	return LocalPackageReference{}, errors.Errorf("%q is not a local package", pkg.PackagePath())
 }
+
+func SortPackageReferencesByPathAndVersion(packages []PackageReference) {
```

Comment:
Is this for future use somewhere? I don't see it used in the main code at the moment, only in the test code.

Reply:
Yeah, I need this when ordering storage packages to create the conversion graph.